packages/core/src/browser/tree/search-box.ts

```diff
@@ -293,6 +291,7 @@ export namespace SearchBox {
 
         export const SEARCH_BOX = 'theia-search-box';
         export const SEARCH_INPUT = 'theia-search-input';
+        export const SEARCH_BUTTONS = 'theia-search-buttons';
```

Comment:
This class name is quite similar to the below, perhaps naming it to `theia-search-buttons-wrapper` could be a little bit more descriptive